scripts/o3de/o3de/manifest.py

```diff
@@ -536,24 +579,29 @@ def get_restricted_json_data(restricted_name: str = None, restricted_path: str o
     if restricted_name and not restricted_path:
         restricted_path = get_registered(restricted_name=restricted_name, project_path=project_path)
 
-    return get_json_data('restricted', restricted_path, validation.valid_o3de_restricted_json)
-
-def get_repo_json_data(repo_uri: str) -> dict or None:
-    if not repo_uri:
-        logger.error('Must specify a Repo Uri.')
+    if not restricted_path:
+        logger.error(f'Restricted Path {restricted_path} has not been registered.')
         return None
 
-    repo_json = get_repo_path(repo_uri=repo_uri)
+    restricted_path = pathlib.Path(restricted_path).resolve()
+    restricted_json = restricted_path / 'restricted.json'
+    if not restricted_json.is_file():
+        logger.error(f'Restricted json {restricted_json} is not present.')
+        return None
+    if not validation.valid_o3de_restricted_json(restricted_json):
+        logger.error(f'Restricted json {restricted_json} is not valid.')
+        return None
 
-    return get_json_data_file(repo_json, "Repo", validation.valid_o3de_repo_json)
+    with restricted_json.open('r') as f:
+        try:
+            restricted_json_data = json.load(f)
+        except json.JSONDecodeError as e:
+            logger.warn(f'{restricted_json} failed to load: {str(e)}')
+        else:
+            return restricted_json_data
 
-def get_repo_path(repo_uri: str, cache_folder: str = None) -> pathlib.Path:
-    if not cache_folder:
-        cache_folder = get_o3de_cache_folder()
+    return None
 
-    repo_manifest = f'{repo_uri}/repo.json'
-    repo_sha256 = hashlib.sha256(repo_manifest.encode())
-    return cache_folder / str(repo_sha256.hexdigest() + '.json')
```

Comment:
Bad Merge here. The `get_repo_json_data` and `get_repo_path` methods have been removed.